packages/api-specification/interface/window.ts

```diff
@@ -117,16 +100,19 @@ declare namespace ssf {
     transparent?: boolean;
   }
 
-  class EventEmitter {
+  /**
+   * Exposes methods that allow subscribing to particular events
+   */
+  abstract class EventEmitter {
```

Comment:
I think other objects are likely to need EventEmitter (e.g. Notification's onClick event), so should this be moved out too?